traitsui/tests/test_editor.py

```diff
@@ -182,7 +184,15 @@ def create_editor(
 
 
 @unittest.skipIf(no_gui_test_assistant, "No GuiTestAssistant")
-class TestEditor(GuiTestAssistant, unittest.TestCase):
+class TestEditor(BaseTestMixin, GuiTestAssistant, unittest.TestCase):
+
+    def setUp(self):
+        BaseTestMixin.setUp(self)
+        GuiTestAssistant.setUp(self)
+
+    def tearDown(self):
+        GuiTestAssistant.tearDown(self)
+        BaseTestMixin.tearDown(self)
```

Comment:
similar comment to before.  Why did we not need to call `GuiTestAssistant.setUp(self)` before but now we do?